web/src/index.tsx

```diff
@@ -1,36 +1,10 @@
-import App from './App'
 import React from 'react'
-import ReactDOM from 'react-dom'
-import { AuthContext, AuthProvider, TAuthConfig } from 'react-oauth2-code-pkce'
-
-const authEnabled = process.env.REACT_APP_AUTH === '1'
-
-const authConfig = {
```

Comment:
Why remove this?
It's an important part of the template imo

Reply:
Added in auth.ts